dhall/ci.dhall

```diff
@@ -24,7 +24,7 @@ let steps =
                 c.Run::{
                 , name = "Test docs"
                 , run = "docs/makeSite"
-                , if = Some "startsWith(matrix.scala, '2.12')"
+                , `if` = Some "startsWith(matrix.scala, '2.12')"
```

Comment:
These backticks are necessary as the previous version was only accepted due to a bug in dhall-haskell.